mlrun/db/httpdb.py

```diff
@@ -987,6 +988,7 @@ def list_artifacts(
         :param producer_uri:    Return artifacts produced by the requested producer URI. Producer URI usually
             points to a run and is used to filter artifacts by the run that produced them when the artifact producer id
             is a workflow id (artifact was created as part of a workflow).
+        :param limit:           Maximum number of artifacts to return. Default is 1000.
```

Comment:
IMO we should document that we do the tag filtering after the limit and you might get less than 1000 artifacts even if you have more with the specific tag.

Reply:
Limit means Maximum - less is accepted 🤷‍♂️

Reply:
what? if you have 2000 artifacts and the 1st 1000 are artifacts with tag v1 and the next 1000 are with v2 and you list with v2 and limit 1000 you will get 0 artifacts. how is that accepted?